FindNearby/scripts/soql/account.soql

```diff
@@ -0,0 +1,6 @@
+// Use .soql files to store SOQL queries.
+// You can execute queries in VS Code by selecting the
+//     query text and running the command:
+//     SFDX: Execute SOQL Query with Currently Selected Text
+
+SELECT Id, Name FROM Account
```

Comment:
Does your project actually need this? if not then you can delete the entire `scripts` folder